cpp/src/groupby/sort/group_sum.cu

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2020, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2021, NVIDIA CORPORATION.
```

Comment:
You should revert this change since the file contents do not appear to have been modified here.

Reply:
```suggestion
 * Copyright (c) 2019-2020, NVIDIA CORPORATION.
```